spec/rubocop/cop/i18n/gettext/decorate_function_message_spec.rb

```diff
@@ -10,41 +10,48 @@
   functions.each do |function|
-     context "#{function} with undecorated double-quote message" do
-       it_behaves_like 'a_detecting_cop', "#{function}(\"a string\")", function, 'should have a decorator around the message'
-       it_behaves_like 'a_fixing_cop', "#{function}(\"a string\")", "#{function}(_(\"a string\"))", function
-       it_behaves_like 'a_no_cop_required', "#{function}(_(\"a string\"))", function
-     end
-     context "#{function} with undecorated single-quoted message" do
-       it_behaves_like 'a_detecting_cop', "#{function}('a string')", function, 'should have a decorator around the message'
-       it_behaves_like 'a_fixing_cop', "#{function}('a string')", "#{function}(_('a string'))", function
-       it_behaves_like 'a_no_cop_required', "#{function}(_('a string'))", function
-     end
-     context "#{function} with undecorated constant & message" do
-       it_behaves_like 'a_detecting_cop', "#{function}(CONSTANT, 'a string')", function, 'should have a decorator around the message'
-       it_behaves_like 'a_fixing_cop', "#{function}(CONSTANT, 'a string')", "#{function}(CONSTANT, _('a string'))", function
-       it_behaves_like 'a_no_cop_required', "#{function}(CONSTANT, _('a string'))", function
-     end
-     context "#{function} with multiline message" do
-       it_behaves_like 'a_detecting_cop', "#{function} 'multi '\\\n 'line'", function, 'should not use a multi-line string'
-     end
-     context "#{function} with concatenated message" do
-       it_behaves_like 'a_detecting_cop', "fail 'this' + 'string' + 'is' + 'concatenated'", function, 'should not use a concatenated string'
-     end
-     context "#{function} with interpolated string" do
-       it_behaves_like 'a_detecting_cop', "#{function}(\"a string \#{var}\")", function, 'interpolation is a sin'
-#      it_behaves_like 'a_fixing_cop', "#{function}(\"a string \#{var}\")", "#{function}(_(\"a string %{value0}\")) % { value0: var, }", function
-       it_behaves_like 'a_no_cop_required', "#{function}(_(\"a string %{value0}\")) % { value0: var, }", function
-     end
-     context "#{function} message not decorated, but does not hit interpolation / concatenation / multi-line / simple-string" do
-       it_behaves_like 'a_detecting_cop', "fail print('kittens')", function, 'There is no decoration'
-     end
-     context "#{function} with the n_ decorator" do
-       it_behaves_like 'a_no_cop_required', "#{function}(n_(\"a string\"))", function
-       it_behaves_like 'a_no_cop_required', "#{function}(n_('a string'))", function
-       it_behaves_like 'a_no_cop_required', "#{function}(CONSTANT, n_('a string'))", function
-       it_behaves_like 'a_no_cop_required', "#{function}(n_(\"a string %{value0}\")) % { value0: var, }", function
-     end
+    context "#{function} with undecorated double-quote message" do
+      it_behaves_like 'a_detecting_cop', "#{function}(\"a string\")", function, 'should have a decorator around the message'
```

Comment:
as in the README, suggest "message should be decorated" or "message should have decoration"